src/services/account.js

```
@@ -39,6 +39,7 @@ const UserTransformDTO = require('@dtos/userDTO')
 const notificationUtils = require('@utils/notification')
 const userHelper = require('@helpers/userHelper')
 const { broadcastEvent } = require('@helpers/eventBroadcasterMain')
+const { use } = require('i18next')
```

ESLint

[error] 'use' is assigned a value but never used.

(no-unused-vars)